src/server/mocked-endpoint.ts

```diff
@@ -20,4 +20,23 @@ export class MockedEndpoint implements MockedEndpointInterface {
         // Wait for all completed running requests to have all their details available
         return Promise.all<CompletedRequest>(this.rule.requests);
     }
+
+    pendingMocks(): string[] {
+        var data = [];
```

Comment:
Generally I'd prefer `const` or `let` everywhere instead of `var` - it's a little clearer about how variables are intended to be used, and it helps you avoid a small but annoying class of errors completely.

Reply:
Ok.